Src/Private/Get-AbrCPPMLicense.ps1

```diff
@@ -39,16 +39,16 @@ function Get-AbrCPPMLicense {
                     $OutObj = @()
                     foreach ($item in $summary) {
                         $OutObj += [pscustomobject]@{
-                            "Type" = $item.license_type
-                            "Count" = $item.licensed_count
-                            "Used" = $item.used_count
+                            "Type"    = $item.license_type
+                            "Count"   = $item.licensed_count
+                            "Used"    = $item.used_count
```

Comment:
Do you have something in your VS Code settings which is changing the alignment?

Reply:
Yes, i use some other project/machine, where it is not alignPropertyValuePairs... and get sometimes issues ^^